src/my-element.ts

```diff
@@ -214,6 +254,37 @@ export class MyElement extends LitElement {
     this._canvasApp()
   }
 
+  private _drawProfileAvatar(angle: number, vector: number, accountId: number) {
+    const avatarSize = 25
+    //This needs some updating for canvas, but otherwise good
+   
+    // const scale = this.height || this.width //scale
+
+    const radians = (angle) * (Math.PI / 180) //-90 orients upwards? So this is radians from the positive y axis?
+
+    const canvas = this._canvas.value as HTMLCanvasElement
+    const ctx = canvas.getContext('2d') as CanvasRenderingContext2D
+    ctx.save()
+    ctx.setTransform(1,0,0,1,0,0) // Something is messing with the transform outside this function, we're resetting it to default before beginning
+    ctx.translate(canvas.width/2, canvas.height/2) //set transform matrix to center of canvas
+    const offset = (this.mapRadius/2) * vector // This gives us the distance from center for our profile image
```

Comment:
So the rotation from center ran into an issue where the images were being rotated as well. Obvi our UI team will be pretty upset with us if all the C and S style folks are looking at their profile pictures upside down.

We're using the same concept to determine placement as the SVG DiSC graph. Determine the offset from the center, use the offset as the radius of a circle, then use Math.sin and Math.cos to determine x and y coords on that circle, draw the profile avatar in that spot.